ExaFunctions/DurableCsvBulkImportFromBlobStorage.cs

```diff
@@ -0,0 +1,184 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Exasol.Utilities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+using MoreLinq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Exasol
+{
+    //http://dontcodetired.com/blog/post/Understanding-Azure-Durable-Functions-Part-4-Passing-Input-To-Orchestrations-and-Activities
+    //https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-create-first-csharp?pivots=code-editor-visualstudio
+    //https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-error-handling?tabs=csharp
+    public static class DurableCsvBulkImportFromBlobStorage
+    {
+        [FunctionName("DurableCsvBulkImportFromBlobStorage")]
+        public static async Task<List<string>> RunOrchestrator(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+
+            ImportOrchestratorParams iop = context.GetInput<ImportOrchestratorParams>();
+            //get the filenames
```

Comment:
I would remove comments like this, your variable names are nice and clear without comments. :) I won't  write about comments anymore, you can check others yourself

Reply:
in this case I feel like they help clarify things because there's quite a bit going on there.